lib/from_honeybee/schedule/ruleset.rb

```diff
@@ -71,6 +71,15 @@ def to_openstudio(openstudio_model)
           day_schedule_new.addValue(time_until, values_day_new[i])
         end
       end
+
+      # assign holiday schedule
+      if @hash[:holiday_schedule]
+        holiday_schedule = openstudio_model.getScheduleDayByName(@hash[:holiday_schedule])
+        unless holiday_schedule.empty?
+          holiday_schedule_object = holiday_schedule.get
+          os_sch_ruleset.setHolidaySchedule(holiday_schedule_object)
+        end
+      end
```

Comment:
Can this be put in a try/except so that it can still work in older OpenStudio versions?

Reply:
@chriswmackey I added a begin rescue statement that should run if the method exists.

Reply:
@chriswmackey The else statement was not necessart in this case.